src/main/webapp/assets/js/utils.js

```diff
@@ -403,6 +403,96 @@ $(document).on('click', '.openOtherInstructorModal', function (e) {
     }).modal('toggle', e);
 });
 
+
+
+// open a modal where user can search for courses with no instructor and select them
+$(document).on('click', '#courseStringTrigger', function (e) {
+  e.preventDefault();
+  var courseString = $.trim($('#courseString').val());
+  var termId = $.trim($('#canvasTermId').text());
+  
+  var mini='/manager/api/v1/accounts/1/courses?search_term=' + courseString + '&per_page=200';
```

Comment:
Is there no way to get this query path from sectiontool.properties so it doesn't need to be repeated here?